cmd/account/mgmt/account-assign.go

```diff
@@ -214,8 +213,18 @@ func (o *accountAssignOptions) findUntaggedAccount(rootOu string) (string, error
 		return "", ErrNoUntaggedAccounts
 	}
 
+	identity, err := o.awsClient.GetCallerIdentity(nil)
```

Comment:
```suggestion
	identity, err := o.awsClient.GetCallerIdentity(&sts.GetCallerIdentityInput{})
```
I think this is more clear, though the behavior should be identical